msg/templates/urtps/RtpsTopics.cpp.template

```diff
@@ -88,7 +89,7 @@ void RtpsTopics::publish(uint8_t topic_ID, char data_buffer[], size_t len)
     switch (topic_ID)
     {
 @[for topic in send_topics]@
-        case @(message_id(topic)): // @(topic)
+        case @(rtps_message_id(os.path.abspath('uorb_rtps_message_ids.json'), topic)): // @(topic)
```

Comment:
This needs to parse a json file for every message. Can we instead pass an already parsed `dict` to the template?

Reply:
Makes sense. Will do that then.